rust/processor/src/db/common/models/object_models/parquet_v2_objects.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::v2_object_utils::{CurrentObjectPK, ObjectAggregatedDataMapping};
+use crate::{
+    db::common::models::default_models::move_resources::MoveResource,
+    schema::current_objects,
+    utils::{database::DbPoolConnection, util::standardize_address},
+};
+use ahash::AHashMap;
+use aptos_protos::transaction::v1::{DeleteResource, WriteResource};
+use bigdecimal::{BigDecimal, ToPrimitive};
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Serialize)]
+pub struct Object {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub object_address: String,
```

Comment:
i forgot to check this with previous loads but did we ensure that addresses are normalized? (0x[0-9A-F]{64})

Reply:
we do for the obejct address in the processor level. 

`SELECT * FROM objects WHERE NOT (object_address LIKE '0x%' AND owner_address LIKE '0x%'); `

this didn't return any unless script is wrong lol

Reply:
no, what i mean by this `LEN(object_address) == 66` should always be true but sometimes during deserialization, leftmost 0s are dropped. For addresses, I haven't  checked but we expect padding to length is always the same. There are edge cases where address is parsed out and needs to be padded

Reply:
/// Standardizes all addresses and table handles to be length 66 (0x-64 length hash)
pub fn standardize_address(handle: &str) -> String {
    if let Some(handle) = handle.strip_prefix("0x") {
        format!("0x{:0>64}", handle)
    } else {
        format!("0x{:0>64}", handle)
    }
}

you mean like this?